package.json

```diff
@@ -92,14 +87,18 @@
     "regenerator-runtime": "0.14.1",
     "semantic-release": "24.2.3",
     "typescript-eslint": "8.28.0",
-    "vinyl-source-stream": "2.0.0"
+    "vite": "6.2.4",
+    "vite-plugin-commonjs": "0.10.4",
+    "vite-plugin-node-polyfills": "0.23.0"
   },
   "optionalDependencies": {
     "crypto-js": "4.2.0"
   },
   "scripts": {
     "build": "node build_releases.js",
     "build:types": "tsc",
+    "build:browser:umd": "cross-env PARSE_BUILD=browser vite build --config vite.config.umd.ts",
+    "build:weapp:umd": "cross-env PARSE_BUILD=weapp vite build --config vite.config.umd.ts",
```

Comment:
Should the script names be simplified by removing the `:umd` suffix, since there are no other suffixes for the same names?

Reply:
The goal is to remove gulp entirely. There will be `:esm`, `:cjs`, `:lib`, `:exports` etc. This is a foundational PR, a POC if you will

Reply:
@mtrezza I removed the prefix